package.json

```diff
@@ -68,14 +70,16 @@
   },
   "scripts": {
     "test": "browserify test/index.js | tape-run | tap-spec",
+    "coverage": "rm -rf coverage && istanbul cover -- tape test/index.js --verbose",
     "validate": "npm ls",
     "start": "run-browser test/index.js",
-    "lint": "jshint --exclude benchmark/ .",
+    "lint": "jshint ampersand-state.js ./test/*",
     "benchmark": "node --allow-natives-syntax benchmark/massCreate.js",
     "preversion": "git checkout master && git pull && npm ls",
     "publish-patch": "npm run preversion && npm version patch && git push origin master --tags && npm publish",
     "publish-minor": "npm run preversion && npm version minor && git push origin master --tags && npm publish",
-    "publish-major": "npm run preversion && npm version major && git push origin master --tags && npm publish"
+    "publish-major": "npm run preversion && npm version major && git push origin master --tags && npm publish",
+    "coveralls": "cat ./coverage/lcov.info | coveralls"
```

Comment:
It's not considered best practice to couple npm scripts. It prevents correct orchestration... In `.travis.yml` I added the `npm run coveralls` as after_scripts as well